backend/problem/views/admin.py

```diff
@@ -143,7 +143,7 @@ def get(self, request):
         response["Content-Length"] = os.path.getsize(file_name)
         return response
 
-    def post(self, request):
+    def post(self, request): 
```

Comment:
줄 마지막에 공백 지워주세요!
> The most important place to avoid adding whitespace is at the end of a line. This is known as trailing whitespace.
> https://realpython.com/python-pep8/